src/utils/command-helpers.mjs

```diff
@@ -174,12 +174,19 @@ export const warn = (message = '') => {
 
 /**
  * throws an error or log it
- * @param {string|Error} message
+ * @param {unknown} message
```

Comment:
It's directly called always in the `try catch` statement where the error is per definition `unknown` as you could throw primitives as well. Therefore the unknown type forces a type guard and therefore narrowing down the type which we are doing here